### PR TITLE
Add color_scheme prop to pc.switch #966

### LINKED ISSUE #966: Add color_scheme prop to pc.switch
We are missing the `color_scheme` prop in pc.switch: https://chakra-ui.com/docs/components/switch/theming

### PR DESCRIPTION
**Pull request by Unknown1934-del**
### All Submissions:

[ ✓] Have you followed the guidelines stated in [CONTRIBUTING.md](https://github.com/pynecone-io/pynecone/blob/main/CONTRIBUTING.md) file?
- [ ✓] Have you checked to ensure there aren't any other open [Pull Requests](https://github.com/pynecone-io/pynecone/pulls ) for the desired changed?

### Type of change

Please delete options that are not relevant.

[ ✓] New feature (non-breaking change which adds functionality)

### New Feature Submission:

[✓ ] Does your submission pass the tests? 
[✓ ] Have you linted your code locally prior to submission?

### Changes To Core Features:

[ x] Have you added an explanation of what your changes do and why you'd like us to include them?
[ x] Have you written new tests for your core changes, as applicable?
[ ✓] Have you successfully ran tests with your changes locally?

### **After** these steps, you're ready to open a pull request.

    a. Give a descriptive title to your PR.
               
               Add color_scheme prop to pc.switch 

    b. Describe your changes.

      In the updated code, I have added a color_scheme variable to the Switch class, which is set to "blue" by default, but can be 
      overridden by passing a color_scheme argument to the constructor. I've also updated the get_style_props method to 
      include the colorScheme property, which is set to the value of self.color_scheme.

    c. Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

Fixes #966

### CODE REVIEW

File: pynecone/components/forms/switch.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 from pynecone.components.component import EVENT_ARG
 from pynecone.components.libs.chakra import ChakraComponent
-from pynecone.vars import Var
+from pynecone.var import Var
```
Comment (picklelo) [inline pynecone/components/forms/switch.py:6]: We recently merged a PR renaming the `var.py` file to `vars.py` so we will need to update this

File: pynecone/components/forms/switch.py [hunk 2]
```diff
@@ -48,3 +51,9 @@ def get_controlled_triggers(cls) -> Dict[str, Var]:
         return {
             "on_change": EVENT_ARG.target.checked,
         }
+        
+    def get_style_props(self) -> Dict:
```
Comment (picklelo) [inline pynecone/components/forms/switch.py:55]: We can delete this function too

**Review by picklelo — COMMENTED** [review]
The PR should be good once we remove the function, we just need to add the new field 🙂

**Review by Unknown1934-del — COMMENTED** [review]
Please tell if this is the correct thing. I am a beginner, I apologize for extending this small issue

**Review by picklelo — APPROVED** [review]
Awesome, thanks for the updates!

**Comment by Unknown1934-del** [comment]
> Awesome, thanks for the updates!

Thank You for Approving !

**Review by picklelo — APPROVED** [review]
Just had to run the linter

**Pull request merged**